content/manuals/engine/network/ca-certs.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Use CA certificates with Docker
+linkTitle: CA certificates
+description: Learn how to install and use CA certificates on the Docker host and in Linux containers
+keywords: docker, networking, ca, certs, host, container, proxy
+---
```

Comment:
Again a feedback from Nike to include a stern warning:

Best practices should be followed (or Consult your security team ) when using MITM CA certificates in production-released containers. If compromised, attackers could intercept sensitive data, spoof a trusted service, or perform man-in-the-middle attacks.